### PR TITLE
Removing underscores ref guide

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

A number of underscores in Reference guide titles, which need to be removed.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2 https://github.com/uyuni-project/uyuni-docs/pull/1095
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/15627

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Reference Guide: removed underscores in page titles and nav bar links.
```
Comment (juliogonzalez) [inline .changelog:7]: Did users see such underscores? If not, the changelog entry is not needed, as it's not relevant for them. Otherwise it can stay.

File: modules/reference/pages/spacecmd/whoamitalkingto.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[[ref-spacecmd-whoamitalkingto]]
```
Comment (juliogonzalez) [inline modules/reference/pages/spacecmd/whoamitalkingto.adoc:1]: How is this related to removing underscores?

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/reference/pages/spacecmd/whoamitalkingto.adoc (hunk 2 above)
Comment (0rnela) [reply modules/reference/pages/spacecmd/whoamitalkingto.adoc:1]: This was in redundant file that was now removed.
